### ISSUE TITLE
Documentation for errors?

### ISSUE THREAD

**Issue by kparal** [flathub/flatpak-builder-lint#31]
I just received the following build error when [trying to update](https://github.com/flathub/org.freefilesync.FreeFileSync/pull/82) FreeFileSync on Flathub:
```
{
    "errors": [
        "finish-args-incorrect-dbus-gvfs",
        "toplevel-cleanup-debug"
    ],
    "warnings": [
        "finish-args-contains-both-x11-and-wayland",
        "flathub-json-deprecated-i386-arch-included"
    ]
}
If you think problems listed above are a false positive, please report it here:
  https://github.com/flathub/flatpak-builder-lint
```

It's great that it contains a link for reporting false positives, but it doesn't contain an obvious link for documentation of those errors. The error IDs don't tell me what the problem is. Where can I find explanation of what they mean and how to best fix them?

Can you please create the documentation (if it doesn't exist already, I can't find it) and provide a link in that error message? A lint tool which is mandatory should at least try to help the packagers understand the problem. Thank you.

**Comment by kparal** [comment]
Thanks a lot for your help :slightly_smiling_face: I'll look at those problems. But that wasn't the purpose of filing this ticket. I believe some packagers will have lots of trouble deciphering what the problem is and how to fix it just by reading the error ID. If there was a documentation (e.g. a wiki page in this project) listing the errors and describing them in a bit more detail (just a few sentences and perhaps a link somewhere else, as you've done, is *much* better than the current state), it would help people a lot (and Flathub as well).

**Comment by hfiguiere** [comment]
There is documentation now

https://docs.flathub.org/docs/for-app-authors/linter/

If something is missing, please submit a PR there.

**Comment by kparal** [comment]
@hfiguiere Great, thanks! Could you also include the link in the linter error output?
